README.md

```diff
@@ -522,6 +522,98 @@ module "function_app_without_build" {
 | archive_file_path | string | Path to the generated archive file |
 | endpoint_url      | string | Endpoint URL                       |
 
+#### MSSQL Database
```

Comment:
The link to this chapter is missing in the table of contents. Perhaps we add `    - [MSSQL Database](#mssql-database)` in line 18 of readme file.

Reply:
good catch!